cache/grpcproxy/BUILD.bazel

```diff
@@ -0,0 +1,41 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "grpcproxy.go",
+        "readcloser.go",
+    ],
+    importpath = "github.com/buchgr/bazel-remote/v2/cache/grpcproxy",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//cache:go_default_library",
+        "//genproto/build/bazel/remote/asset/v1:go_default_library",
+        "//genproto/build/bazel/remote/execution/v2:go_default_library",
+        "//utils/backendproxy:go_default_library",
```

Comment:
I think this code is missing from your PR...

Reply:
This is existing code, it is already used by all the proxy implementations

Reply:
D'oh ;)